src/test/java/upgrade/com/atlassian/bitbucket/jenkins/internal/SafeRemovedClassList.java

```diff
@@ -0,0 +1,33 @@
+package upgrade.com.atlassian.bitbucket.jenkins.internal;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SafeRemovedClassList {
```

Comment:
Do I understand correctly that the safe removed class list is for classes that we've authored, have changes that break the test but we are confident will not impact the upgrade flow?

Reply:
Pretty much, it is for changes that cannot be annotated (say a removal of a class) that you've manually tested and deemed safe for upgrade.